rules/cloud/aws/cloudtrail/aws_cloudtrail_console_login_failure.yml

```diff
@@ -0,0 +1,24 @@
+title: AWS ConsoleLogin Failure Monitoring
+id: 6393e346-1977-46ef-8987-ad414a145fad
+status: test
```

Comment:
All new rules start with the "experimental" status so please change them accordingly